packages/venia-ui/lib/targets/venia-ui-intercept.js

```diff
@@ -60,4 +60,18 @@ module.exports = veniaTargets => {
     });
 
     new CategoryListProductAttributes(venia);
+
```

Comment:
Since we're inside of venia, the import and usage should be added directly to the footer.js file. No need for targetables.